users/faq.rst

```diff
@@ -315,6 +315,29 @@ protect against unauthorized access. Either:
 
 In all cases, username/password authentication and HTTPS should be used.
 
+I don't like the GUI / Theme. Can it be changed?
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+You can change the theme in the settings. Syncthing ships with a default and
```

Comment:
> As of 0.14.10, there is also a black theme, which is the old dark theme with black background.

I think this is just a bit confusing, and will be useless in a couple of month's time. Why not say it ships with other themes, and leave it at that?